channels/rdpdr/client/rdpdr_main.c

```diff
@@ -1170,7 +1171,10 @@ static UINT rdpdr_send_client_name_request(rdpdrPlugin* rdpdr)
 	WINPR_ASSERT(rdpdr);
 
 	if (!rdpdr->computerName[0])
-		gethostname(rdpdr->computerName, sizeof(rdpdr->computerName) - 1);
+	{
+		DWORD size = sizeof(rdpdr->computerName) - 1;
+		GetComputerNameA(rdpdr->computerName, &size);
```

Comment:
Not your code (and not the scope of this PR), but I know that some companies working on proxies would be interested if this `GetComputerNameA` could be abstracted so that the returned name can be controlled (doesn't it make some problems with our proxy impl ?)

Reply:
@hardening not if you just use `passthrough` mode for `rdpdr` (then this path is not used), also with `intercept` this function is not used (see `rdpdr_send_client_name_request` in `pf_channels_rdpdr.c`